src/D2L.Bmx/BmxConfigProvider.cs

```diff
@@ -18,7 +22,7 @@ internal class BmxConfigProvider( FileIniDataParser parser ) : IBmxConfigProvide
 				var tempdata = parser.ReadFile( configFileName );
 				data.Merge( tempdata );
 			} catch( Exception ) {
-				Console.Error.Write( $"WARNING: Failed to load the global config file {configFileName}." );
+				consoleWriter.WriteWarning( $"WARNING: Failed to load the global config file {configFileName}." );
```

Comment:
It's hard to reproduce such a failure scenario so it's never manifested, but `Console.Error.Write` (without a newline) would've never worked nicely.